test/unit/browser/invalidation_mixin_test.dart

```diff
@@ -49,12 +49,12 @@ void main() {
         Future onValidation = thing.invalidate();
 
         // ignore: unawaited_futures
-        onValidation.then(expectAsync((ValidationStatus status) {
+        onValidation.then(expectAsync1((ValidationStatus status) {
           expect(status, equals(ValidationStatus.complete));
-        }, count: 1));
+        }));
 
         // ignore: STRONG_MODE_DOWN_CAST_COMPOSITE
-        thing.onValidate.listen(expectAsync((_) {}, count: 1));
+        thing.onValidate.listen(expectAsync1((_) {}));
```

Comment:
are you able to get rid of the `// ignore: STRONG_MODE_DOWN_CAST_COMPOSITE` with this change?